scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -0,0 +1,1542 @@
+from __future__ import (absolute_import)
+import os
+try:
+    from mantidplot import *
+    canMantidPlot = True
+except ImportError:
+    canMantidPlot = False
+
+from . import ui_sans_data_processor_window as ui_sans_data_processor_window
+from PyQt4 import QtGui, QtCore
+from mantid.simpleapi import *
```

Comment:
Is this used? I would assume that mantid stuff would have been called by the presenter?